src/build_runner/master.zig

```diff
@@ -69,15 +69,15 @@ pub fn main() !void {
     cache.addPrefix(global_cache_directory);
     cache.hash.addBytes(builtin.zig_version_string);
 
-    const host = try std.zig.system.NativeTargetInfo.detect(.{});
+    const host = try std.zig.system.resolveTargetQuery(.{});
```

Comment:
This doesn't match what I have found in zig's [build_runner.zig](https://github.com/ziglang/zig/blob/04ac028a2cd20ea430dd6a05b956363ec182b6ff/lib/build_runner.zig#L49-L52). `query` appears to store the parameter that has been passed to `resolveTargetQuery`.
```suggestion
    const host: std.Build.ResolvedTarget = .{
        .query = .{},
        .result = try std.zig.system.resolveTargetQuery(.{}),
    };
```
The call to `std.Build.create` will also need to be adjusted.